modules/space/translation/keplertranslation.cpp

```diff
@@ -198,10 +198,12 @@ KeplerTranslation::KeplerTranslation(const ghoul::Dictionary& dictionary)
     setKeplerElements(
         p.eccentricity,
         p.semiMajorAxis,
-        p.inclination,
-        p.ascendingNode,
-        p.argumentOfPeriapsis,
-        p.meanAnomaly,
+        p.inclination < 0.0 ? p.inclination + 360.0 : p.inclination,
```

Comment:
How about:
`p.inclination < 0.0 ? (fmod(p.inclination, 360.0) + 360.0) : p.inclination`
in case the negative value is less than -360, since the `codegen::inrange` won't guard against such a value being read?

Reply:
The codegen checks against `[[codegen::inrange(-360.0, 360.0)]];` so < -360 values can't happen

Reply:
I tested an input file with an angle greater than 360.0 and it didn't throw any errors, so that's why I think the < -360 value could happen.

Reply:
Oh... gotcha. Will investigate

Reply:
I couldn't reprocude it;  loading these asset files correctly produce errors for all the fail ones:

[2784.zip](https://github.com/OpenSpace/OpenSpace/files/11949890/2784.zip)

Reply:
Sorry for the confusion, I had been testing with the `readSbdbFile` function in kepler.cpp, where the angles outside the range of -360 to 360 are allowed, and do work. Anyway, I withdraw my suggestion.

Reply:
No worries!  Ended up finding a different bug while tracking this down ;)